src/routing/core/connectivity.rs

```diff
@@ -134,7 +134,9 @@ impl Core {
             return Ok(vec![]);
         }
 
-        self.propose_offline(name)
+        let mut commands = self.propose_offline(name)?;
```

Comment:
Shouldn't we `propose_offline` only after we did `StartConnectivityTest` and considered it offline?

Reply:
This call is within `handle_peer_lost` so it's _after_ we know that the peer is offline. The signal is for other nodes to start the connectivity test